### ISSUE TITLE
Why BN/Scale layers are not included in fine-tune?

### ISSUE THREAD

**Issue by ltqusst** [Orpine/py-R-FCN#116]
I notice that all batchnorm and scale layers from backbone are not to be fine-tuned, why?
